src/main/kotlin/com/bridgecrew/utils/UIUtils.kt

```diff
@@ -1,14 +1,15 @@
 package com.bridgecrew.utils
```

Comment:
perhaps not the place but - it should be `io.bridgecrew` because it need to follow the domain, no?

Reply:
that's following the namespace of the project, and it should reflect a unique name (that's usually the domain name). I guess it should be io.bridgecrew but it's not affecting us. If we want we can change this in the future I'm sure we'll want to make some refactors once we're done.